### DISCUSSION TITLE
Does NEW Point blank work with the i-FILTER@Cloud Agent extension without problems?

### DISCUSSION THREAD

**Discussion by gamelist1990** [3kh0/ext-remover#546] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Help me

**Comment by fnfgo** [comment, marked as answer]
Maybe but most likely yes

**Reply by gamelist1990** [reply]
I don't think there is a block page for i-FILTER@Cloud Agent as far as I know, but is there actually?

**Comment by gamelist1990** [comment]
Extensions for i-FILTER@Cloud Agent ↓

https://chrome.google.com/webstore/detail/i-filtercloud-agent/bkfmfndedcalgdgaklieojjfbbjpmijd

**Comment by Aka-but-nice** [comment]
> Help me

Are you literally from the 3kh0 discord, if so, you asked this question which I've answered with no.
